src/index.js

```diff
@@ -67,18 +88,32 @@ class LndEngine {
   }
 
   /**
-   * Validates the current engine
+   * Validates and sets the current state of an engine
    *
    * @returns {void}
    */
   async validateEngine () {
     try {
-      // We do not await this function because we want the validations to run in the background.
-      // It can take time for the engines to be ready, so we use exponential backoff to retry validation
-      // for a period of time, until it is either successful or there is actually something wrong.
-      const validationCall = () => this.validateNodeConfig()
       const payload = { symbol: this.symbol }
       const errorMessage = 'Engine failed to validate. Retrying'
+      const validationCall = async () => {
+        // We make a initial call to check if engine is unlocked. The result updates
+        // `this.unlocked` and is then used in the underlying configuration check.
+        this.unlocked = await this.isEngineUnlocked()
```

Comment:
I don't think we need to save the state of the `unlocked` status since we don't use it outside of this call and instead depend on the engine's overall validation status.

Reply:
@treygriffith as a note, this may be needed, but can be added in a later PR. The broker _could_ call `isEngineUnlocked` separately, but we would probably want to "cache" that information instead of continually making an RPC call